test/scripts/ledger_fixer.test.js.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Not sure how this file ended up here? Fyi, a common practice to run tests on this API (since the full suite is pretty slow) is to use:

```bash
npm run test -- test/server/graphql/v2/mutation/TierMutations.test.ts
```

Or for TDD:

```bash
npm run test -- --watch test/server/graphql/v2/mutation/TierMutations.test.ts
```